Dockerfile

```diff
@@ -1,20 +1,23 @@
-ARG NODE_IMAGE=node:12.2-alpine
+ARG NODE_IMAGE=node:10.16-alpine
```

Comment:
Looks like v12 will be LTS in a month, do you really think this is worth changing? https://nodejs.org/en/about/releases/

Reply:
Yeah agree, I would not suggest downgrading the Node version if all the code has been testing/working with v12 plus it's still the "current" version of NodeJS.

Reply:
I agree with you, i did not know the life cycle of node.js. I will setup the curent stable `ARG NODE_IMAGE=node:12.11-alpine`